tt1-blocks/templates/page.html

```diff
@@ -1,29 +1,67 @@
 <!-- wp:template-part {"slug":"header","tagName":"header","className":"site-header"} /-->
 
 <!-- wp:group {"tagName":"main"} -->
-<main class="wp-block-group">
```

Comment:
Let's separate the changes into different PRs so they can be reviewed individually